src/main/java/de/tum/in/www1/artemis/config/migration/MigrationRegistry.java

```diff
@@ -6,13 +6,17 @@
 import java.util.TreeMap;
 
 import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.annotation.Profile;
 import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Component;
 
+import de.tum.in.www1.artemis.config.migration.entries.MigrationEntry20230808_203400;
+
 /**
  * This component allows registering certain entries containing functionality that gets executed on application startup. The entries must extend {@link MigrationEntry}.
  */
 @Component
+@Profile("scheduling")
```

Comment:
Because we only need to execute it once. Not n times, which could create conflicts in the startup process.
Requested by Stephan.

Reply:
So, we only added this to make sure the migration is executed only on a single instance and it has nothing to do with scheduling? If so, could you please explain it with a comment because I get the rationale behind it but without comment it really isn't inituitive.